packages/apollo-server-express/src/ApolloServer.test.ts

```diff
@@ -253,5 +256,81 @@ describe('apollo-server-express', () => {
         });
       });
     });
+    describe('file uploads', () => {
+      it.only('enabled uploads', async () => {
+        server = new ApolloServer({
+          typeDefs: gql`
+            scalar Upload
```

Comment:
Needing to add this line is frustrating. When it is removed, the `makeExecutableSchema` in the constructor fails. We could delay the evaluation of the typeDefs and resolvers until later when `this.schema` is used with a getter. This feels a bit weird, since the mocks are added in the constructor, so could or could not affect the typeDefs later added to the schema with `enhanceShema`

Before the current and past two commits, we had the upload configuration code in the ApolloServer constructor. However fileuploads seem to be a integration specific concern, so I'm pretty convinced that uploads should be configured in `registerServer`